fixtures/samplePlugin.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (data) => {
+  return function cssModules(tree) {
+    data.resourceId = tree.options.rollupResourceId;
```

Comment:
We adhere to generally accepted conventions. In this case, it is better to use `from`

Reply:
What particular conventions do you mean? I can't find any neither https://rollupjs.org/guide/en/#conventions nor https://github.com/posthtml/posthtml/blob/master/docs/plugins/guide.md